client/src/app/__tests__/KeyboardBindingsSpec.js

```diff
@@ -248,6 +248,44 @@ describe('KeyboardBindings', function() {
   });
 
 
+  it('removeSelection', function() {
+
+    // given
+    event = createKeyEvent('Backspace');
+
+    keyboardBindings.update([{
+      accelerator: 'Delete',
+      action: 'removeSelection'
+    }]);
+
+    // when
+    keyboardBindings._keyDownHandler(event);
+
+    // then
+    expect(actionSpy).to.have.been.calledWith(null, 'removeSelection');
+  });
+
+
+  it('removeSelection - mac', function() {
+
+    // given
+    event = createKeyEvent('Delete');
+
+    keyboardBindings.setIsMac(true);

```

Comment:
Why don't we create keyboardBindings with appropriate options for mac test? There is no other place we use this setter.

Reply:
Makes sense to pass it during construction. Alternative is to make it a private API, i.e. `_setMac(true|false)`.

Reply:
Makes sense though.